sdk/src/main/scala/com/horizen/account/history/AccountHistory.scala

```diff
@@ -9,7 +9,7 @@ import com.horizen.account.storage.AccountHistoryStorage
 import com.horizen.consensus._
 import com.horizen.params.{NetworkParams, NetworkParamsUtils}
 import com.horizen.validation.{HistoryBlockValidator, SemanticBlockValidator}
-import scorex.util.{ModifierId, ScorexLogging}
+import scorex.util.{ModifierId, ScorexEncoding, ScorexLogging}
```

Comment:
ScorexEncoding is a part of scorex-util project, which is still kept as is in sparkz 2.0.0-RC8.
Will be replaces in the one of the releases.

Reply:
Ok, but I have found that there is a sparkz.core.utils.SparkzEncoding and a com.horizen.SparkzEncoding, in addition to ScorexEncoding and they seem very similar to each other. There is a rule when to use one instead of another?

Reply:
@i-skrypnyk pls take a look if it is expected or require a further fix/cleanup on sparkz/sdk sides

Reply:
Thanks @saratnt, @i-Alex. This is indeed some redundant duplication, that should be removed. Will do as part of the current sparkz update.